projects/sunbird-video-player/src/lib/components/video-player/video-player.component.ts

```diff
@@ -216,12 +206,13 @@ export class VideoPlayerComponent implements AfterViewInit, OnDestroy {
     if (this.player) {
       this.player.dispose();
     }
-    this.unlistenTargetMouseEnter();
-    this.unlistenTargetMouseLeave();
-    this.unlistenControlDivMouseEnter();
-    this.unlistenControlDivMouseLeave();
-    this.unlistenControlDivTouchEnd();
-    this.unlistenControlDivTouchStart();
+    this.unlistenTargetMouseMove();
     this.unlistenTargetTouchStart();
   }
+
+  checkDevice() {
```

Comment:
Use code from https://www.npmjs.com/package/mobile-device-detect, not the package to detect if it is mobile or not

Reply:
Move this method to service